Ical.Net/CalendarComponents/VTimeZone.cs

```diff
@@ -293,8 +292,8 @@ public VTimeZone(string tzId) : this()
         Location = _nodaZone.Id;
     }
 
-    private DateTimeZone _nodaZone;
-    private string _tzId;
+    private DateTimeZone _nodaZone = DateTimeZone.Utc; // must initialize
```

Comment:
In a future (v6) version, the Noda tz should not be part of this model object at all.